static/src/javascripts/projects/common/modules/navigation/header-top-nav.js

```diff
@@ -0,0 +1,555 @@
+import debounce from 'lodash/debounce';
```

Comment:
Guessing most of this is copied from `new-header.js` which is actually the old header? :)

Reply:
Yes, exactly! Not confusing at all.